draft-ietf-trans-gossip.md

```diff
@@ -984,19 +984,22 @@ auditor.
 
 ## Interaction
 
-The interactions of the mechanisms is thus outlined:
+Assuming no other log consistency measures exist, clients who perform
+only a subset of the mechanisms described in this document are exposed
+to the following vulnerabilities:
 
 HTTPS clients can be attacked without risk of detection if they do not
-participate in any of the three mechanisms.
+participate in any of the three mechanisms (or any other, unspecified
+mechanisms).
```

Comment:
_Any_ other is not technically true. I suggest removing the part within parentheses -- that type of "future proofing" seems unnecessary.